src/main/java/gregtech/api/creativetab/GTCreativeTabs.java

```diff
@@ -0,0 +1,33 @@
+package gregtech.api.creativetab;
```

Comment:
Should these even still be in the api package at all? I don't really see a need for them to be, and so they most likely can be moved to `common`

Reply:
I agree that these should be in common